README.md

```diff
@@ -1,5 +1,6 @@
 # demo-firebase-ios
 A simple iOS application that demonstrates how end-to-end encryption works with firebase as a backend service for authentication and chat messaging. While this is a chat app, you can reuse it in any other apps to protect user data, documents, images.
+You can also watch [this YouTube video](https://www.youtube.com/watch?v=6zpzbcm_3I8) to see the whole tutorial.
```

Comment:
I think, this is not grammatically correct to write "*watch video to see the tutorial*" construction. Some sort of tautology. Maybe you should write "*watch video with the tutorial*" or "*click [here](https://www.youtube.com/watch?v=6zpzbcm_3I8) to see tutorial*".